stable/datadog/templates/daemonset.yaml

```diff
@@ -218,6 +218,14 @@ spec:
           - name: pointerdir
             mountPath: /opt/datadog-agent/run
           {{- end }}
+          {{- if (and (.Values.datadog.logsEnabled) (.Values.datadog.logsConfigContainerCollectAll)) }}
+          - name: logpodpath
```

Comment:
Do we need both mounts? We can use another runtime than docker, so `/var/lib/docker/containers` wouldn't be relevant in this case.